src/frequenz/sdk/actor/power_distributing/_component_managers/_pv_inverter_manager/_pv_inverter_manager.py

```diff
@@ -169,6 +179,15 @@ async def distribute_power(self, request: Request) -> None:
         )
         await self._set_api_power(request, allocations, remaining_power)
 
+    async def _set_to_zero_if_not_working(self) -> None:
+        if not self._component_status_rx:
+            return
+        async for status in self._component_status_rx:
+            if status.component_id in self._pv_inverter_ids:
+                if status.value == ComponentStatusEnum.NOT_WORKING:
+                    api_client = connection_manager.get().api_client
+                    await api_client.set_power(status.component_id, 0.0)
```

Comment:
It turns out this was a hack to handle the special case where there's data, but not enough for the resampler.  And this was actually never triggered, because according to the power distributor, the inverters always had data.

This also conflicts with our other approach, to assume that inverters with missing data are at full power or min power based on the safety requirements of applications.  So I've dropped this commit from this PR.